typebinder/src/exporters/file.rs

```diff
@@ -42,13 +45,35 @@ impl FileExporter {
                 .expect("Invalid UTF-8 name for module")
         });
     }
+
+    pub fn set_header_comment(&mut self, header_comment: HeaderComment) {
+        self.header_comment = header_comment;
+    }
+
+    fn render_header_comment(&self, rust_module_path: &syn::Path) -> Option<String> {
+        match &self.header_comment {
+            HeaderComment::None => None,
+            HeaderComment::Custom(comment) => Some(format!("// {}", comment)),
+            HeaderComment::Standard => {
+                let header = format!(
+                    "// This file is auto generated with typebinder from rust source code. Do not change this file manually.\n\
+                     // Change the rust source code instead and regenerate with typebinder.\n\
+                     // Relevant rust module: {}",
```

Comment:
"Rust module" (capital R), 

I would have written "Rust source module : {}" but that's irrelevant